composer.json

```diff
@@ -17,7 +17,7 @@
     "ext-dom": "*",
     "ext-libxml": "*",
     "apimatic/core-interfaces": "^0.1.0",
-    "apimatic/jsonmapper": "^3.0.4",
+    "apimatic/jsonmapper": "dev-37-oaf-discriminator-support",
```

Comment:
This will be reverted after releasing: https://github.com/apimatic/jsonmapper/pull/38

Reply:
Updated to ^3.1.1